README.md

```diff
@@ -33,6 +33,10 @@ The runtime has a built-in command to determine if your host system is capable o
 $ cc-runtime cc-check
 ```
 
+## Quick start for users
+
+See the installation guides available for various operating systems [here](docs/).
```

Comment:
So that we can avoid "here", lets move the link to "installation guides" as has been done in the two sections following this one:
See the [installation guides](docs/) available for various operating systems.

Reply:
Fixed.